nvflare/fuel/f3/drivers/aio_http_driver.py

```diff
@@ -103,7 +103,7 @@ def supported_transports() -> List[str]:
 
     @staticmethod
     def capabilities() -> Dict[str, Any]:
-        return {DriverCap.HEARTBEAT.value: True, DriverCap.SUPPORT_SSL.value: True}
+        return {DriverCap.HEARTBEAT.value: False, DriverCap.SUPPORT_SSL.value: True}
```

Comment:
Why is "DriverCap.HEARTBEAT.value: False"?
Who is sending the heartbeat now?

Reply:
The HeartbeatMonitor in SFM is sending the heartbeat now.